src/test_pages/TimelinePage.ts

```diff
@@ -38,8 +38,6 @@ export default class TimelinePage extends BasePage {
   }
 
   public async verifyRouteSelectionVisible(isVisible: boolean) {
-    isVisible
-      ? await expect(this.route_select).toBeVisible({ timeout: 2000 })
-      : await expect(this.route_select).toBeHidden({ timeout: 2000 })
+    await this.verifySelectionVisible(this.route_select, isVisible)
```

Comment:
Hi! I suggest you not to mix between the tickets. If you write tests for the single line page, you shouldn't think this time about anything except this page.

Reply:
I will keep this in mind in future PRs, I've changed it back to how it was before to avoid conflicts with the other tickets.
I'm open to further feedback and would be happy to make additional adjustments if needed.

Thank you!